pkgs/tools/misc/zoxide/default.nix

```diff
@@ -27,22 +20,7 @@ rustPlatform.buildRustPackage rec {
 
   buildInputs = lib.optionals stdenv.isDarwin [ libiconv ];
 
-  # tests are broken on darwin
-  doCheck = !stdenv.isDarwin;
-
-  # fish needs a writable HOME for whatever reason
-  preCheck = ''
-    export HOME=$(mktemp -d)
-  '';
-
-  checkInputs = [
-    fish
-    powershell
-    shellcheck
-    shfmt
-    xonsh
-    zsh
-  ];
+  doCheck = true;
 
```

Comment:
`doCheck` defaults to `true` :)

```suggestion
```

Reply:
@cole-h thanks, fixed! Btw, do you know why `libiconv` is needed on macOS?

Reply:
No clue, I don't use macOS.